Makefile

```diff
@@ -83,10 +84,20 @@ fallthrough: submodules
 
 go.test.unit: $(KUBEBUILDER)
 
+# Generate required functions for CRD structs to be treated as runtime.Object
+# When we move the cloud providers out-of-tree, only the first call will be needed.
+generate: vendor
```

Comment:
I believe all of these generation commands can be handled as `//go:generate` comments in `pkg/apis/apis.go`

Reply:
agreed, we should not need a specific Makefile target added that runs the golang generation, that should all be driven by the `go.generate` target in [`golang.mk`](https://github.com/upbound/build/blob/master/makelib/golang.mk#L250) and the `//go:generate` directive that @hasheddan pointed out in [`pkg/apis/apis.go`](https://github.com/crossplaneio/crossplane/blob/master/pkg/apis/apis.go#L17-L18)